LibreNMS/Alert/Transport/Alerta.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/*Copyright (c) 2019 GitStoph <https://github.com/GitStoph>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details. */
+ /**
+ * API Transport
+ * @author GitStoph <https://github.com/GitStoph>
+ * @copyright 2019 GitStoph
+ * @license GPL
+ * @package LibreNMS
+ * @subpackage Alerts
+ */
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+
+class Alerta extends Transport
+{
+    public function deliverAlert($obj, $opts)
+    {
+        if (empty($this->config)) {
```

Comment:
In here, you test if $this->config is empty, and still run the exact same code as outside the condition.

Reply:
Whoops! forgot to clean that up. Thanks!